### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by fschlueter**
This PR adds unit tests for the Response class. So far only the multiply operator functionality is tests ....

Based on PR #635. Solves #642

### CODE REVIEW

**Comment by fschlueter** [comment]
@sjoerd-bouma

**Comment by fschlueter** [comment]
ping

File: NuRadioReco/detector/test/tests.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
-python3 NuRadioReco/detector/test/test_rnog_detector.py
\ No newline at end of file
+python3 NuRadioReco/detector/test/test_rnog_detector.py
+python3 NuRadioReco/detector/test/test_response.py
```
Comment (sjoerd-bouma) [inline NuRadioReco/detector/test/tests.sh:5]: I think this won't work reliably - if the first python script raises an error, the second one will probably still run, and the github test will only consider the output of the second test. Feel free to put this into a new test, I think.

**Review by sjoerd-bouma — COMMENTED** [review]
See comment, otherwise this is probably fine : )

File: NuRadioReco/detector/test/tests.sh (hunk 1 above)
Comment (fschlueter) [reply NuRadioReco/detector/test/tests.sh:5]: I see thanks! I fixed it in the workflow yaml. Would that work?

**Comment by fschlueter** [comment]
ping

**Comment by fschlueter** [comment]
@cg-laser maybe you can approve this?

**Review by sjoerd-bouma — APPROVED** [review]
Oops! Should have approved this a long time ago. Thank Felix!

**Pull request merged**